packages/components/CallToActionSection.tsx

```diff
@@ -13,28 +13,31 @@ export default function CallToActionSection() {
   const { t } = useI18n("call-to-action");
 
   return (
-    <Section className="py-40">
-      <Stack textAlign="center" align="center" spacing={{ base: 8, md: 10 }}>
-        <Heading fontWeight={600} fontSize={{ base: "3xl", sm: "4xl" }} lineHeight="110%">
+    <Section className="py-40 px-8">
+      <div className="text-center flex flex-col items-center gap-6">
+        <h2 className="font-semibold leading-tight text-3xl sm:text-4xl">
           <Trans
             i18nKey={t("title")}
             components={{
               span: <span className="text-purple-400" />,
             }}
           />
-        </Heading>
+        </h2>
 
         <p className="text-gray-500 max-w-3xl">{t("description")}</p>
 
-        <Stack spacing={6} w={{ base: "100%", sm: "auto" }} direction={{ base: "column", sm: "row" }}>
-          <Button as="a" href={links.secondaryButton} rounded="full" px={6}>
+        <div className="w-full sm:w-auto flex flex-col sm:flex-row">
+          <a
+            href={links.secondaryButton}
+            className="rounded-full px-6 py-2 font-semibold bg-gray-100 text-black hover:bg-gray-200"
```

Comment:
Let's use Daisy UI button for adding styles here:
- https://daisyui.com/components/button/